docs/settings.md

```diff
@@ -0,0 +1,92 @@
+```{eval-rst}
+:tocdepth: 3
+```
+
+```{currentModule} segy.config
+
+```
+
+# Settings Management
+
+```{article-info}
+:author: Altay Sansal
+:date: "{sub-ref}`today`"
+:read-time: "{sub-ref}`wordcount-minutes` min read"
+:class-container: sd-p-0 sd-outline-muted sd-rounded-3 sd-font-weight-light
+```
+
+## `SegyFileSettings` Class
+
+The [SegyFileSettings] is a configuration object for the
+[SegyFile] in the environment. It allows you to customize various aspects of
+SEG-Y file parsing according to your needs and the specifics of your project.
+
+It is composed of various sub-settings isolated by SEG-Y components and various topics.
+
+- **BINARY**: The [SegyBinaryHeaderSettings] is used for binary header configuration
+  while reading a SEG-Y file.
+- **ENDIAN**: This setting determines the byte order that is being used in the SEG-Y file.
+  The possible options are `"big"` or `"little"` based on [Endianness]. If left as None,
+  the system defaults to Big Endian (`"big"`).
+- **REVISION**: This setting is used to specify the SEG-Y revision number. If left as
+  None, the system will automatically use the revision mentioned in the SEG-Y file.
+- **USE_PANDAS**: This setting is a boolean that decides whether to use pandas for
```

Comment:
Maybe we emphasize that this only applies to the Trace Headers and not the Trace Data. Also we don't at the moment support turning this setting off.

Reply:
good point i'll fix